python/morphology.cpp

```diff
@@ -188,6 +199,8 @@ void register_morphology(pybind11::module& m) {
                 "The relative position of the proximal end of the cable on its branch ∈ [0,1].")
         .def_readonly("dist", &arb::mcable::dist_pos,
                 "The relative position of the distal end of the cable on its branch ∈ [0,1].")
+        .def("__eq__",
```

Comment:
If you are overloading an operator that is defined directly in C++ (as is the case here):
```
.def(pybind11::self == pybind11::self)
```
See: https://github.com/pybind/pybind11/blob/master/tests/test_operator_overloading.cpp
You'll need to include `pybind11/operators.hpp`

Reply:
The code that's there does work though — is there a reason to use the `operators.hpp` way instead, other than being a bit shorter?

Reply:
It's just shorter. I pointed it out, because I don't think that it is in the docs, so you might not have been aware of it.
It is a bit clearer, and is consistent with changes I recently made in another PR that implemented operator overloading for `cv_policy`.